cpp/tests/bitmask/bitmask_tests.cpp

```diff
@@ -575,12 +574,13 @@ TEST_F(CopyBitmaskTest, TestZeroOffset)
   for (auto& m : validity_bit) {
     m = this->generate();
   }
-  auto input_mask = cudf::test::detail::make_null_mask(validity_bit.begin(), validity_bit.end());
+  auto input_mask =
+    std::get<0>(cudf::test::detail::make_null_mask(validity_bit.begin(), validity_bit.end()));
 
   int begin_bit         = 0;
   int end_bit           = 800;
-  auto gold_splice_mask = cudf::test::detail::make_null_mask(validity_bit.begin() + begin_bit,
-                                                             validity_bit.begin() + end_bit);
+  auto gold_splice_mask = std::get<0>(cudf::test::detail::make_null_mask(
```

Comment:
Our of curiosity, is there a reason you prefer `std::get<0>` to `.first` for pairs? For consistency with tuples?

Reply:
I like that this gets us the first value from the structured binding without an unused variable and without resorting to the vagueness of `.first` and `.second`.